odl/__init__.py

```diff
@@ -16,18 +16,16 @@
 # along with ODL.  If not, see <http://www.gnu.org/licenses/>.
 
 
-"""ODL is a functional analysis library with a focus on discretization.
+"""ODL (Operator Discretization Library)
 
-ODL suppors abstract sets, linear vector spaces defined on such
-and Operators/Functionals defined on these sets. It is intended
-to be used to write general code and faciliate code reuse.
+ODL is a Python library for fast prototyping focusing on (but not
+restricted to) inverse problems.
 """
 
 from __future__ import absolute_import
 
 __version__ = '0.5.4.dev0'
-__all__ = ('diagnostics', 'discr', 'operator', 'set', 'space', 'solvers',
-           'tomo', 'trafos', 'util', 'phantom', 'deform', 'ufunc_ops')
+__all__ = ()
```

Comment:
Hm, right. But that basically means that everything has to stay, since the same applies to the other subpackages. Currently you get everything from the subpackages' `__all__` but the other stuff is not accessible, at least not directly.

That also means that we have to be careful not to include any module or package into `__all__` that conflicts with another name somewhere in the library, or at least make sure that they're not imported into the same namespace.